storybook/stories/API/props/AnimationProps.ts

```diff
@@ -15,20 +15,12 @@ export const animationBegin = {
   },
 };
 export const animationDuration = {
-  description: 'Specifies the duration of animation, the unit of this option is ms.',
-  type: { name: 'number' },
-  defaultValue: 1500,
```

Comment:
Funny enough, this was misleading. It was correct for some components, but not for others.